server/lobby.js

```diff
@@ -694,7 +688,9 @@ class Lobby {
 
     onNodeReconnected(nodeName, games) {
         _.each(games, game => {
-            let syncGame = new PendingGame({ username: game.owner }, { spectators: game.allowSpectators, name: game.name });
+            let ownerUser = game.players[game.owner];
```

Comment:
Wouldn't normally comment on naming, but this is a bit confusing when followed by `ownerUser.user`. Really, this is a player object.